.github/workflows/image-builder.yml

```
@@ -0,0 +1,70 @@
+# Build a docker image on push to main
+# Also tag the image if commit is tagged
+
+name: "Image Builder"
+
+on:
+  release:
+    types:
+      - "created"
+      - "released"
+  push:
+    branches:
+      - "main"
+    paths:
+      - "**.rs"
+      - "**.toml"
+      - "**.yml"
+      - "**.lock"
+      - "genesis/**"
+      - "Dockerfile"
+
+jobs:
+  build-image:
+    # only run if prechecks pass and the source branch is named 'prerelease/*'
+    runs-on: [self-hosted, linux]
+    env:
+      IMAGE_NAME: "cennznet/depositClaim"
```

convention for image names it `-`
```suggestion
      IMAGE_NAME: "cennznet/bridge-relayer-service"
```